AirCasting/Utils/StringsCustomizer.swift

```diff
@@ -0,0 +1,61 @@
+// Created by Lunar on 07/02/2022.
+//
+
+import SwiftUI
+
+final class StringCustomizer {
+    static func customizeString(_ s: String, using words: [String], fontWeight: Font.Weight = .bold, color: Color = .aircastingGray, font: Font = .body, makeNewLineAfterCustomized: Bool = false) -> Text {
+        
+        var customizedText: Text!
```

Comment:
Why "!" ?

Reply:
Otherwise, we would get an error, and using unwrapping we are ensuring compiler that customizedText will have a value in the future

Reply:
But if it doesn't then the app crashes and the compiler doesn't notify us about that. I thought generally it's better not to use the force unwrap

Reply:
Exactly, I was going to propose the same.